packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1152,8 +1161,10 @@ class _RenderDecoration extends RenderBox {
           break;
       }
       final double dy = lerpDouble(0.0, floatingY - labelOffset.dy, t);
+      final double yOffset = decoration.labelAlignment == TextFieldLabelAlignment.top
```

Comment:
I looked for a better place to put this, but I couldn't find where `label` has its `dy` calculated.  Any idea where that is or is there a better place?